docs/pages/careers/react-engineer-docs-infra.md

```diff
@@ -0,0 +1,113 @@
+
+## Why we're hiring
+
+Documentation is arguably one the most essential pieces of any of our products.
+It's how developers learn about each corner of a given library, becoming empowered to bend them to the needs of their projects.
+It also plays as an illustration of the principles we hold true to the product. It needs to be performative, fast, beautiful, and clear.
+
+Over the last two years, we have established an internal docs-infra team responsible for pushing the documentation platform forward, developing features such as automatically-generated OG images, multi-tab component demos, speed performance improvements, and many others.
+
+With that in mind, we're looking for someone to own the documentation as a product and become the first full-time dedicated person to it.
+We want to make it significantly more mature so that many other features (including those with revenue generation capacity) can come to life.
+
+## The role
+
+The Docs Infra team currently doesn't have anyone dedicated to it full-time.
+So, as the first React engineer to be primarily focused on it, you will work on developing new features and infrastructure-related tasks that will, ultimately, make documentation generation and consumption smoother, better, and faster.
+
+This role involves touching on a piece of MUI's infrastructure that all of our products depend on, so you will have frequent opportunities to interact with the whole company.
+On a day-to-day basis, it will be a very dynamic role, as you'll have tasks that range from being more operational to user-facing.
+You will also play a fundamental role in defining the Docs Infra roadmap and priorities.
+
+### Key responsibilities
+
+- Ensure the documentation feels fast and snappy for readers.
+  You'll be closely keeping tabs on Lighthouse scores and bundle size
+- Develop new features that will enhance the way the libraries are showcased
+- Improve content findability by refining the documentation search quality
+- Ensure SEO and accessibility are strong and stable
+- Improve the documentation authoring experience for both the maintainers and the community
+- Own any infrastructure-related tasks, like managing dependencies, analytics, TypeScript modules, and hosting allowing each product team to never worry about the documentation platform and be 100% focused on their product
+
+### Tasks and initiatives you might take on
+
+- Migrate the docs platform to Next.js App Router
+- Allow users to search the docs within a specific product scope
+- Enable to store different parts of our documentation in different repositories
+- Improve the blog experience by introducing new elements like a CMS
+- Standardize the documentation design and layout across all of MUI's products
+- Enhance our code blocks with ref
+- Introduce a more scalable solution for hosting videos
+- Introduce a CDN for managing images
```

Comment:
I looked at GitHub issues, PRs, and shaping pages to get some of these examples. They seem to be illustrative of the main things, but let me know if there's anything we should include!